### ISSUE TITLE
Give users the option to tooltip on focus

### ISSUE THREAD

**Issue by ConorMurphy21** [ConorMurphy21/StrikeOrSike#183]
* For accessibility reasons, tooltips should be accessible to people who don't use a mouse
